tests/util/pool_tests.rs

```diff
@@ -61,6 +68,78 @@ pub fn test_thinpool_expand(paths: &[&Path]) -> () {
 }
 
 
+/// Verify a snapshot has the same files and same contents as the origin.
+pub fn test_filesystem_snapshot(paths: &[&Path]) {
+    let dm = DM::new().unwrap();
+    let (mut pool, _) =
+        StratPool::initialize("stratis_test_pool", &dm, paths, Redundancy::NONE, true).unwrap();
+    let &(_, fs_uuid) = pool.create_filesystems(&[("stratis_test_filesystem", None)])
+        .unwrap()
+        .first()
+        .unwrap();
+    let write_buf = &[8u8; SECTOR_SIZE];
+    let file_count = 10;
+    let source_tmp_dir = TempDir::new("stratis_testing").unwrap();
+    {
+        // to allow mutable borrow of pool
+        let filesystem = pool.get_mut_strat_filesystem(&fs_uuid).unwrap();
```

Comment:
I don't think it's necessary to use ```get_mut_strat_filesystem()```, probably immutable will do.

Reply:
ok